commons/commons-lang/src/test/java/com/clicktravel/common/security/SanitiserTest.java

```diff
@@ -91,4 +91,53 @@ public void shouldSanitise_withApostropheInValue() {
         // Then
         assertEquals("O'Test", sanitisedValue);
     }
+
+    // Below tests for sanitiseIfMarkupPresent method
```

Comment:
I don't think we need this comment and can instead update the test names to reflect what method is being tested, e.g. 

```
shouldNotSanitiseIfMarkupPresent_whenNoMarkUp
```